### PR TITLE
vim-patch.sh, vim-patch:bc8801c9317eb

### PR DESCRIPTION
**Pull request by justinmk**

### CODE REVIEW

File: scripts/vim-patch.sh [hunk 1]
```diff
@@ -136,9 +136,12 @@ preprocess_patch() {
...
' +w +q "$file"
 
   # Remove channel.txt, netbeans.txt, os_*.txt, todo.txt, version*.txt, tags
-  local na_doc='channel\.txt\|netbeans\.txt\|os_\w\+\.txt\|todo\.txt\|version\d\.txt\|tags$'
+  local na_doc='channel\.txt\|netbeans\.txt\|os_\w\+\.txt\|todo\.txt\|version\d\.txt\|tags'
   2>/dev/null $nvim --cmd 'set dir=/tmp' +'g@^diff --git a/runtime/doc/\%('${na_doc}'\)@norm! d/\v(^diff)|%$
' +w +q "$file"
 
+  # Remove "Last change ..." changes in doc files.
+  2>/dev/null $nvim --cmd 'set dir=/tmp' +'%s/^@@.*\n.*For Vim version.*Last change.*\n.*For Vim version.*Last change.*//' +w +q "$file"
```
Comment (justinmk) [inline scripts/vim-patch.sh]: Helps avoid lots of noise when merging docs.

Comment (jamessan) [inline scripts/vim-patch.sh]: Except now the diff is malformed, and these were simple changes to apply.

Comment (justinmk) [inline scripts/vim-patch.sh]: `patch -p1` seems to apply the diff ok. I will try to correct the diff, but in general we need to remove noise from the patches, it adds friction.

Comment (jamessan) [inline scripts/vim-patch.sh]: I would agree if we didn't have those lines in the files at all, but now this just leads to outdated information.

Comment (justinmk) [inline scripts/vim-patch.sh]: But now we can replace the headers with a simple Nvim title. No date.

Comment (justinmk) [inline scripts/vim-patch.sh]: #6328

**Pull request merged**